src/components/metrics/card/MetricCardViewSection/index.js

```diff
@@ -113,14 +114,14 @@ export class MetricCardViewSection extends Component {
         }
 
         <div className='MetricTokenSection'>
```

Comment:
I would make a separate 'showMetricToken' boolean and use that to filter the rendering of this component, instead of dumping all of that logic in the sub-component.